packages/modeling/src/measurements/measureBoundingSphere.js

```diff
@@ -0,0 +1,146 @@
+const flatten = require('../utils/flatten')
+
+const vec2 = require('../maths/vec2')
+const vec3 = require('../maths/vec3')
+
+const geom2 = require('../geometries/geom2')
+const geom3 = require('../geometries/geom3')
+const path2 = require('../geometries/path2')
+const poly3 = require('../geometries/poly3')
+
+const cacheOfBoundingSpheres = new WeakMap()
+
+/*
+ * Measure the bounding sphere of the given (path2) geometry.
+ * @return {[[x, y, z], radius]} the bounding sphere for the geometry
+ */
+const measureBoundingSphereOfPath2 = (geometry) => {
```

Comment:
ok, so I am not convinced that this is doing what it implies.  I agree that it calculates a bounding sphere for the geometry, but I don't believe that it calculates the smallest bounding sphere for the geometry.

imagine a cloud of points: 
```
    x          x
      x   x      
 x                                                      x
     x        x
```
The centroid would be heavily weighted to the left, the radius would be big enough to encompass the right-most point, and so the bounding sphere would include a bunch of empty space on the left.

Reply:
It’s definitely NOT the most efficient sphere. That calculation takes huge amounts of parallel computations based on simulation. This is just a “rough” estimate, but still valid.

Reply:
I tend to trust choices from threejs :)